src/main.ts

```diff
@@ -657,12 +661,23 @@ export async function getToken(input: CspInput): Promise<string> {
 export async function loadAllData(executionGraph: Object): Promise<string[]> {
   let files: string[] = []
 
+  const onlyUploadOnFailure = core.getInput("upload-only-on-failure")
```

Comment:
The name of the input property seems wrong. According to your other changes above, it should be `only-upload-on-failure`